pkg/config/configdefaults/config_default.go

```diff
@@ -0,0 +1,67 @@
+package configdefaults
+
+import (
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/library-go/pkg/crypto"
+)
+
+func DefaultString(target *string, defaultVal string) {
+	if len(*target) == 0 {
```

Comment:
How about nil check as well?

Reply:
> How about nil check as well?

There's nothing you can do to fix the call at that point.  We discussed it in the original pull and decided it was programmer error.